charts/rollups-validator-node/values.yaml

```
@@ -4,76 +4,81 @@
 ## dapp deployment paramaters
 ##
 dapp:
-  ## dapp.image is a complete image name, e.g. "cartesi/dapp:echo-python-0.8.0-server"
-  ## REQUIRED
+  # -- dapp.image is a complete image name, e.g. "cartesi/dapp:echo-python-0.8.0-server"
```

we can remove the parameter name from the description to avoid the repetition in the generated readme

```suggestion
  # -- a complete image name, e.g. "cartesi/dapp:echo-python-0.9.1-server"
```

---

do the same for every other parameter

---

Fixed !